### PR TITLE
System Information button for CASP

### PR DESCRIPTION
**Pull request by mvidner**
https://trello.com/c/XDgl2dj7
https://fate.suse.com/322328

Main PR: https://github.com/yast/yast-installation/pull/487

### CODE REVIEW

**Comment by mvidner** [comment]
This is such a trivial widget, I thought about keeping it in yast2-installation, but in the end this is cleaner.

File: src/lib/tune/widgets.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# current contact information at www.suse.com.
+# ------------------------------------------------------------------------------
+
+require "yast"
+require "cwm/widget"
+
+module Tune
+  module Widgets
+    class SystemInformation < CWM::PushButton
+      include Yast::Logger
```
Comment (jreidinger) [inline src/lib/tune/widgets.rb]: all widgets by default have logger integrated, so not needed ;)

File: src/lib/tune/widgets.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        textdomain "tune"
+      end
+
+      def label
+        # PushButton label
+        _("System Information")
+      end
+
+      def handle
+        Yast::WFM.CallFunction("inst_hwinfo", [])
```
Comment (jreidinger) [inline src/lib/tune/widgets.rb]: if it can change values, then it have to return :redraw symbol, otherwise it won't be properly refreshed.

Comment (mvidner) [inline src/lib/tune/widgets.rb]: @lslezak AFAIK the System Information dialog cannot change things that are displayed in the installation proposal / all-in-one overview, can it?

Comment (lslezak) [inline src/lib/tune/widgets.rb]: The module can internally change some kernel settings, but AFAIK it does not report anything in the summary.

**Pull request merged**